src/main/java/com/teamsfinder/userwriteservice/user/model/User.java

```diff
@@ -14,29 +16,39 @@
 import javax.persistence.JoinColumn;
 import javax.persistence.JoinTable;
 import javax.persistence.ManyToMany;
+ import javax.persistence.Table;
+ import javax.validation.constraints.NotEmpty;
  import javax.validation.constraints.NotNull;
+ import java.util.ArrayList;
  import java.util.List;
 import java.util.UUID;
 
 @Entity
 @Getter
 @Setter
-class User {
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "user_profile")
+public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @NotNull
-    private UUID keyCloakId;
+    @NotEmpty
+    private String keyCloakId;
     @NotNull
     @Enumerated(EnumType.STRING)
-    private AccountType accountType;
+    @Builder.Default
+    private AccountType accountType = AccountType.USER;
     private String githubProfileUrl;
     private String profilePictureUrl;
+    private boolean blocked;
+    @Builder.Default
     @ManyToMany
     @JoinTable(
             name = "user_tags",
             joinColumns = @JoinColumn(name = "user_id"),
             inverseJoinColumns = @JoinColumn(name = "tag_id")
     )
-    private List<Tag> tags;
+    private List<Tag> tags = new ArrayList<>();
```

Comment:
To new tu chyba niepotrzebne

Reply:
https://stackoverflow.com/questions/20715143/to-initialize-or-not-initialize-jpa-relationship-mappings

Reply:
Jak zostawie bez tego to w builderze stworzy mi immutable list a potrzebuje mutable